dotnet/samples/GettingStartedWithProcesses/Step03/Step03a_FoodPreparation.cs

```diff
@@ -128,4 +230,63 @@ protected async Task UsePrepareSpecificProductAsync(ProcessBuilder processBuilde
         });
         Console.WriteLine($"=== End SK Process '{processBuilder.Name}' ===");
     }
+
+    // Path used for storing json processes samples in repository
+    private readonly string _currentSourceDir = Path.Combine(
```

Comment:
nit: I think most of the methods below this point could be moved into another file to help with the readability of the sample. They are needed by the sample but maybe not contributing to the concepts the sample is teaching.